lib/semantics/check-omp-structure.cc

```diff
@@ -30,52 +31,51 @@ bool OmpStructureChecker::HasInvalidCloselyNestedRegion(
   return false;
 }
 
-void OmpStructureChecker::CheckAllowed(const OmpClauseEnum &type) {
+void OmpStructureChecker::CheckAllowed(const OmpClause &type) {
   if (ompContext_.empty()) {
```

Comment:
Are these checks for an empty context stack still necessary now that you push a context even when there is an error?

Reply:
You're right, I missed these checks.